pwnlib/commandline/libcdb.py

```diff
@@ -158,14 +179,39 @@ def print_libc(libc):
     for symbol in libc['symbols'].items():
         log.indented('\t%25s = %s', symbol[0], symbol[1])
 
-def handle_remote_libc(args, libc):
-    print_libc(libc)
-    if args.download_libc:
-        path = libcdb.search_by_build_id(libc['buildid'], args.unstrip)
-        if path:
-            if args.unstrip:
-                libcdb.unstrip_libc(path)
```

Comment:
Removing this causes cached libcs to not be unstripped even when `unstrip` was set.

Reply:
It will unstrip in `search_by_hash`
https://github.com/Gallopsled/pwntools/blob/b08f4b7bc8eaf37a6f77a1255e6d8462dd5c3e14/pwnlib/libcdb.py#L283-L285

Reply:
Sorry, I overlooked that. But why not unstrip before `return cache`?

Reply:
True, that's the better place to put this!